README.md

```diff
@@ -1,6 +1,10 @@
 # NGRX-Traits
 
-NGRX Traits is a set of factory functions that dynamically generate actions, effects, reducers needed for common functionality like loading a list of entities, filtering, pagination, sorting. They can be mix and match, so you only use what you need, and you can create your own to for example reuse some business logic across your app.
+<!-- I was thinking in explaining what is a trait is  -->
+<!-- Do you think that forth mentioning the factory functions? or some synonim that people that reads this won't have to go and make sure what is a factory  -->
+
+NGRX Traits is a set of factory functions that dynamically generate actions, effects, reducers needed for common functionality like loading a list of entities, filtering, pagination, sorting, etc. They can be mixed and matched, so you only use what you need, and you can create your own too, for example, reuse some business logic across your app.
```

Comment:
Im thinking maybe just removing the factory  an dynamically words maybe the generate is enouguh,  so it will read like `is a set of functions that dynamically generate actions, effects, reducers needed for common functionality...`